include/conf/load.hpp

```diff
@@ -55,6 +89,25 @@ namespace color_coded
       while(std::getline(ifs, tmp))
       { args.emplace_back(detail::make_absolute(std::move(tmp), base)); }
 
+      return args;
+    }
+
+    inline args_t load(std::string const &file, std::string const &filetype, std::string const &fileName = "")
```

Comment:
>Please don't use default arguments; they're a maintanence nightmare.

Would you care to elaborate? I do not see how this is any different from a default argument.
Maybe we can directly call load_color_coded in this case?
```c++
inline args_t load(std::string const &file, std::string const &filetype)
{ return load_color_coded(file, filetype); }
```
> Aside from that, please remove the camelCase from `fileName`.

Done. Will be included in the next push.